README.md

```diff
@@ -1,6 +1,6 @@
 # Terraform Provider Hashicups
 
-This repo is a companion repo to the [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers) Learn collection. 
+This repo is a companion repo to the [Call APIs with Terraform Providers](https://developer.hashicorp.com/terraform/tutorials/providers) tutorials. 
```

Comment:
Please note that there is a new framework-specific collection now: https://learn.hashicorp.com/collections/terraform/providers-plugin-framework

Reply:
```suggestion
This repo is a companion repo to the [Call APIs with Terraform Providers](https://learn.hashicorp.com/collections/terraform/providers-plugin-framework) tutorials. 
```